README.md

```diff
@@ -373,6 +374,18 @@ Developers actively use version control tools for a long time, probably since 20
 
 </article>
 
+<article id="development-browsers">
+    
+### Development Browsers
+
+Development browsers have features that help developers and designers create and test websites and apps.
+
+* [Blisk](https://blisk.co) — provides a development workspace to develop and test modern web applications twice faster. 
```

Comment:
I think the correct link is https://blisk.io/

Reply:
You are right :flushed: I'll update it right away.